projects/plugins/boost/app/assets/src/js/pages/settings/sections/Score.svelte

```diff
@@ -102,6 +104,14 @@
 	function dismissModal() {
 		modalData = null;
 	}
+
+	const panelStore = performanceHistoryPanelDS.store;
+	const onTogglePerformanceHistory = status => {
+		panelStore.set( status );
+	};
+
+	$: performanceHistoryNeedsUpgrade = $modulesState.performance_history.available === false;
+	$: performanceHistoryIsOpen = $panelStore;
```

Comment:
Should this be bubbled further up? Seems like this component now has more stores than it had before.

Reply:
I've left that for the second pass.